arbnode/node.go

```diff
@@ -697,11 +703,7 @@ func createNodeImpl(
 		if dp != nil {
 			stakerAddr = dp.Sender()
 		}
-		whitelisted, err := stakerObj.IsWhitelisted(ctx)
-		if err != nil {
-			return nil, err
-		}
-		log.Info("running as validator", "txSender", stakerAddr, "actingAsWallet", wallet.Address(), "whitelisted", whitelisted, "strategy", config.Staker.Strategy)
+		log.Info("running as validator", "txSender", stakerAddr, "actingAsWallet", wallet.Address(), "strategy", config.Staker.Strategy)
```

Comment:
Maybe move the log.Info to be done by the staker during Initialize()?
That way legacy could print could print if it's whitelisted, bold could print whatever is relevant to it.

Reply:
Done.